Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py

```diff
@@ -205,33 +206,34 @@ def PyExec(self):
 
                 if self._vanadium_ws:
                     van_ws = mtd[self._vanadium_ws[index]]
+                    van_ws_name = self._vanadium_ws[index]
```

Comment:
This is mostly there.

We should move this line up and use it to assign to `van_ws`:
```python
van_ws_name = self._vanadium_ws[index]
van_ws = mtd[van_ws_name]
```